test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -162,10 +162,9 @@ void test_explicit_move(InputIterator i, InputIterator j) {
 struct test_zip_iterator {
     template <typename T1, typename T2>
     void operator()(::std::vector<T1>& in1, ::std::vector<T2>& in2) {
-        //test that zip_iterator is default constructible
-        oneapi::dpl::zip_iterator<decltype(in1.begin()), decltype(in2.begin())> b;
+        static_assert(::std::is_default_constructible<oneapi::dpl::zip_iterator<decltype(in1.begin()), decltype(in2.begin())>>::value,"Zip_iterator isn't default constructible");
```

Comment:
The line is too long. Please split it on two. For example the bool expression on the one line and the string literal on another line

Reply:
Also please change the line to "zip_iterator is not default constructible"